src/sfctl/commands.py

```diff
@@ -246,6 +250,36 @@ def load_command_table(self, args):  # pylint: disable=too-many-statements
             group.command('get', 'get_property_info')
             group.command('delete', 'delete_property')
 
+        # ---------------
+        # Mesh standard commands
+        # ---------------
+        with CommandGroup(self, 'mesh gateway', mesh_gateway_func_path,
+                          client_factory=mesh_gateway_create) as group:
+            group.command('show', 'get')
+            group.command('delete', 'delete')
+            group.command('list', 'list')
+
+        with CommandGroup(self, 'mesh network', mesh_network_func_path,
+                          client_factory=mesh_network_create) as group:
+            group.command('show', 'get')
+            group.command('delete', 'delete')
+            group.command('list', 'list')
+
+        with CommandGroup(self, 'mesh code-package', mesh_code_package_func_path,
+                          client_factory=mesh_code_package_create) as group:
+            group.command('show', 'get_container_logs')
+
+        with CommandGroup(self, 'mesh secret', mesh_secret_func_path,
+                          client_factory=mesh_secret_create) as group:
+            group.command('show', 'get')
+            group.command('delete', 'delete')
+            group.command('list', 'list')
+
+        with CommandGroup(self, 'mesh secretvalue', mesh_secret_value_func_path,
```

Comment:
Should secretvalue be two words?

Reply:
This came up with sfmesh and the consensus was that it would have been weird/ feel inconsistent there to have the command group split into 2 words